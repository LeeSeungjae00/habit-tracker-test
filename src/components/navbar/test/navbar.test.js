import React from 'react';
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer'
import Navbar from '../navbar';

describe('navber', () => {
    const totalCount = 10;
    let navbarComponent;

    beforeEach(() => {
        navbarComponent = (<Navbar totalCount={totalCount} />);
    })
    it('renders', () => {
        const component = renderer.create();
        expect(component.toJSON).toMatchSnapshot();
    })

    it('totalCount is show navbar count', () => {
        render(navbarComponent);
        const span = screen.getAllByText(totalCount);
        expect(span).toHaveLength(1);
    })
})