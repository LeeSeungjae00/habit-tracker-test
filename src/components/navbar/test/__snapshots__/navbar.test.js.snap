// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navber renders 1`] = `[Function]`;
